backend/src/Command/SuperAdmin/SuperAdminCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Command\SuperAdmin;
+
+use App\Request\Admin\AdminRegisterRequest;
+use App\Service\Admin\AdminServiceInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class SuperAdminCommand extends Command
+{
+    protected static $defaultName = 'app:create-super-admin';
+
+    private AdminServiceInterface $adminService;
+
+    public function __construct(AdminServiceInterface $adminService)
+    {
+        $this->adminService = $adminService;
+
+        parent::__construct();
+    }
+
+    public function superAdminData(): AdminRegisterRequest
+    {
+        $adminRegisterRequest = new AdminRegisterRequest();
```

Comment:
since we hard-coded the data that we wanna save, no need for the Request class. but it is fine for now u can keep it.